SPECS/crash-gcore-command/crash-gcore-command.spec

```diff
@@ -41,10 +41,11 @@ install -pm 755 gcore.so %{buildroot}%{_libdir}/crash/extensions/
 %doc COPYING
 
 %changelog
-*   Thu Dec 16 2021 Pawel Winogrodzki <pawelwi@microsoft.com> - 1.5.1-2
+*   Fri Mar 04 2022 Andrew Phelps <anphel@microsoft.com> 1.6.1-1
+-   Update to version 1.6.1
+*   Thu Dec 16 2021 Pawel Winogrodzki <pawelwi@microsoft.com> 1.5.1-2
```

Comment:
```suggestion
*   Thu Dec 16 2021 Pawel Winogrodzki <pawelwi@microsoft.com> - 1.5.1-2
```

Reply:
Keeping this as-is only because it matches all of the existing changelog entries